SqlDataCatalog-OneTrust/ExportTo-OneTrust.ps1

```diff
@@ -0,0 +1,182 @@
+        -Method PUT `
+        -Headers $script:oneTrustApiHeaders `
+        -ContentType 'application/json' `
+        -Body $body
+    
+    return ($response.Content | ConvertFrom-Json).data.id  
+}
+function script:upsertAssets($instanceId, $instanceName, $databaseId, $databaseName, $tags) {
+    If (-not ($createdInstances.ContaInsKey($instanceId))) {
+        $assetId = script:upsertAsset -id $instanceId -name $instanceName -type 'Instance'
+        $createdInstances.Add($instanceId, $assetId)
+    }
+    $instanceAssetId = $createdInstances[$instanceId]
+
+    $databaseAssetId = script:upsertAsset `
+        -id $databaseId `
+        -name $databaseName `
+        -type 'Database'
+
+    script:associate -databaseId $databaseAssetId -withInstanceId $instanceAssetId
+    script:linkDataElements -databaseName $databaseName -assetId $databaseAssetId -tags $tags
+}
+#endregion
+
+Write-Host "SQL Data Catalog API Server: $ServerUrl"
+Write-Host "OneTrust API Server: https://$OneTrustHostName/api"
+
+#region Load Redgate Data Catalog Module
+$moduleName = 'RedgateDataCatalog.psm1'
+If (-not $UseCachedModule -or -not (Test-Path $moduleName)) {
+    Invoke-WebRequest -Uri "$ServerUrl/powershell" -OutFile $moduleName `
+        -Headers @{ 'Authorization' = "Bearer $AuthToken" }
+}
+Import-Module ".\$ModuleName"
+#endregion
+
+Connect-SqlDataCatalog -ServerUrl $ServerUrl -AuthToken $AuthToken
+
+$databases = Get-ClassificationInstance | 
+    ForEach-Object { Get-ClassificationDatabase -InstanceName $_.Name } |
+    Where-Object { -not ($ExcludedDatabaseNames -contains $_.name) }
+    
+$tagCategories = Get-ClassificationTaxonomy | Select-Object -ExpandProperty "TagCategories"
+$DataSubjectId = $tagCategories | Where-Object { $_.name -eq $DataSubject } | Select-Object -ExpandProperty "id"
+$DataElementsId = $tagCategories | Where-Object { $_.name -eq $DataElements } | Select-Object -ExpandProperty "id"
+
+$databases | ForEach-Object {
+    $database = $_
+
+    $columns = Get-ClassificationColumn `
+        -InstanceName $database.instanceName `
+        -DatabaseName $database.name
+
+    $columnTagsToExport = $columns | Where-Object { $_.tags.categoryId -eq $DataSubjectId }
+    $export = ($columnTagsToExport | Select-Object  -Property  @{
+            l = $DataSubject;
+            e = {
+                ($_.tags | 
+                    Where-Object { $_.categoryId -eq $DataSubjectId } | 
+                    Sort-Object "name" | 
+                    Select-Object -ExpandProperty "name") -join "|"
+            } 
+        },
+        @{
+            l = $DataElements;
+            e = { $_.tags | Where-Object { $_.categoryId -eq $DataElementsId } } 
+        }) |
+    Group-Object $DataSubject | 
+    Select-Object -Property @{
+        l = $DATA_SUBJECT
+        e = { $_.Name } 
+    }, @{
+        l = $DATA_ELEMENT
+        e = { ($_.Group.$DataElements.name | Select-Object -Unique | Sort-Object) -join "|" } 
```

Comment:
We could leave it as an array instead of `|` separated values, this way we don't have to split by `|` to get the array.
```suggestion
        e = { ($_.Group.$DataElements.name | Select-Object -Unique | Sort-Object) } 
```